docs/modelserving/v1beta1/triton/torchscript/README.md

```diff
@@ -336,34 +340,38 @@ The transformer does not enforce a specific schema like predictor but the genera
 ```
 
 ```bash
+# download the input file
+curl -O https://raw.githubusercontent.com/kserve/kserve/release-0.8/docs/samples/v1beta1/triton/torchscript/image.json
```

Comment:
TODO: Need to fix this.
Currently, there is no `image.json` file properly working in any release branch.

Reply:
I changed it downloads files from the master branch of KServe repo.
([3f50fa2](https://github.com/kserve/website/pull/127/commits/3f50fa2bde4d50c9aceff754d52b92c41aed6cc7))